ProcessMaker/Repositories/CaseRepository.php

```diff
@@ -10,6 +10,11 @@
 
 class CaseRepository implements CaseRepositoryInterface
 {
+    /**
+     * @var CaseStarted|null
```

Comment:
Please add a comment describing why reference to a $this->case is needed

Reply:
Comment was added